ui/app/AppLayouts/Chat/popups/community/CreateCommunityPopup.qml

```diff
@@ -63,11 +63,9 @@ StatusModal {
         }
     ]
 
-    StackLayout {
+    contentItem: StackLayout {
```

Comment:
not sure what you mean, since it's set to follow it's parent width and height (root/popup ids) it will resize together with it. also not sure why and how you get this behavior shown in the video. Here's what I see

https://user-images.githubusercontent.com/31625338/172348351-d0b9ba65-29a0-481e-836c-18093c64f5cf.mov

Reply:
not sure what you mean, since it's set to follow it's parent width and height (root/popup ids) it will resize together with it. also not sure why and how you get this behavior shown in the video. Here's what I see

https://user-images.githubusercontent.com/31625338/172348351-d0b9ba65-29a0-481e-836c-18093c64f5cf.mov

Reply:
alright, thanks for the second try :) all changed to parent instead of root/popup

Reply:
https://github.com/status-im/status-desktop/pull/6010#discussion_r892127314 - that's a good summary of what I wanted to explain here! Would be great if the [contribution guideline](https://github.com/status-im/StatusQ/blob/master/src/contributing.md) is extended with these two points :crossed_fingers:

Reply:
> I will add another point here I think that the `StatusModal` paddings should be implemented like this:
> 
> ```js
> padding: 16
> topPadding: padding + headerImpl.implicitHeight
> bottomPadding: padding + footerImpl.implicitHeight
> leftPadding: padding
> rightPadding: padding
> ```
> 
> This way you can simply change the contents padding outside the `StatusModal` by changing it's `padding` property.

Good initiative, but if the client sets `topPadding` or `bottomPadding` to some custom value it would still be broken. IMO `StatusModal` should be reworked completely. It's wrong to put header and footer in the background item as is done now, because of the implications:

1) broken content padding - clients can't set padding in a convenient way
```
    StatusModal {
        header.title: "Random dialog"
        topPadding: 16 // it should be top padding of the content

        Text {
            anchors.fill: parent
            text: "Header makes me invisible"
        }
    }
```

2) some artificial and strange properties required, like: `advancedHeaderComponent`, `advancedFooterComponent`
```
    StatusModal {
        showFooter: false
        showAdvancedFooter: true
        advancedFooterComponent: CustomFooter {}
    }
```
should simply be:
```
    StatusModal {
        footer: CustomFooter {}
    }
```

By rework I mean that we should create two separate components, that is: 
- `StatusDialog` which derives from Qt's `Dialog`, no need for custom code for header and footer as it's already covered by the base component
- `StatusPopup` which derives from Qt's `Popup`, used in the scenarios where "dialog" is not needed

I will create issue from that comment (it was on my todo list for long time already) :sweat_smile: also I tried to discuss this topic on Discord before but nobody followed :man_shrugging:

Reply:
Issue created: https://github.com/status-im/StatusQ/issues/720